tests/client/core/test_middleware.js

```diff
@@ -88,6 +88,18 @@ describe('Prefix Middleware', () => {
     assert.sameMembers(fakeRes.set.firstCall.args[1], ['accept-language', 'user-agent']);
   });
 
+  it('should lookup app based on ua string', () => {
```

Comment:
nit of all nits: "look up..." because it's not used as a noun here

![Uploading debbie-downer.gif…]()